servers/services/src/main/java/org/projectnessie/services/authz/ServerAccessContext.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (C) 2020 Dremio
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.projectnessie.services.authz;
+
+import java.security.Principal;
+import javax.annotation.Nullable;
+import org.immutables.value.Value;
+
+@Value.Immutable(prehash = true)
+public abstract class ServerAccessContext implements AccessContext {
+
+  @Override
+  public abstract String operationId();
```

Comment:
nit: we have previously been keeping with the `getOperationId` syntax for getters

Reply:
that's just how it was named in the original design doc in https://github.com/nastra/nessie/blob/be631849f0c0a75b6f36ffd728b461e7586b1e1f/design/authorization.md#L76. I don't have a preference of whether it should have a `get` prefix or not